frontend/src/components/main/surveys-list.test.tsx

```diff
@@ -69,4 +69,23 @@ describe('Surveys list', () => {
       })
     );
   });
+
+  it('not should show a survey when api return error', async () => {
```

Comment:
```suggestion
  it('should not show a survey when api return error', async () => {
```

Reply:
@mizek1 , you are welcome to contribute as well, please consider working on some issues ❤️

Reply:
I will @edimossilva 😄